.changeset/itchy-chicken-cry.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
The changeset is empty.

Reply:
That was intentional. There is no changeset needed and GitHub complained that there was no changeset, suggesting to create an empty changeset if no change log entry was needed. Maybe I misunderstood that message. I'll verify.